### ISSUE TITLE
Problem to extract chictopia.sql.gz

### ISSUE THREAD

**Issue by BrunoKrinski** [cad0p/maskrcnn-modanet#21]
I'm having problems to extract chictopia.sql.gz

gzip: chictopia.sql.gz: not in gzip format

How to solve it?

**Comment by cad0p** [comment]
Thy this and let me know

https://itsfoss.com/how-solve-stdin-gzip-format/

**Comment by BrunoKrinski** [comment]
Hi, thanks for the reply. Well, it didn't worked

tar xvf chictopia.sql.gz

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now

**Comment by cad0p** [comment]
Have you tried deleting the folder and redoing the script again?

Just making sure since it used to work very well

**Comment by BrunoKrinski** [comment]
Yes. I deleted and downloaded the dataset again and it had the same problem. I tried to follow the steps described here: https://github.com/kyamagu/paperdoll/tree/master/data/chictopia

And also, tried to access dataset here: http://vision.is.tohoku.ac.jp/~kyamagu/research/paperdoll/

However, it seems that the links are broken.

**Comment by BrunoKrinski** [comment]
with the following commands :

wget https://github.com/kyamagu/paperdoll/raw/master/data/chictopia/chictopia.sql.gz
gunzip -c chictopia.sql.gz | sqlite3 chictopia.sqlite3

provided here: https://github.com/hrsma2i/dataset-PaperDoll

I succeeded.

**Comment by cad0p** [comment]
I think the issue you had was related to not having git LFS installed,
because the cloning process did not go correctly.

If you look at this line:

https://github.com/cad0p/maskrcnn-modanet/blob/master/maskrcnn_modanet/download.sh#L51

You can see that the following command is exactly like the one you
suggested here.

Anyways, it's good to know that the issue is resolved!
